packages/vite/src/node/server/send.ts

```diff
@@ -17,6 +17,8 @@ const alias: Record<string, string | undefined> = {
   json: 'application/json',
 }
 
+const sourcemapRe = /^\/\/# sourceMappingURL=.+/m
```

Comment:
There's `mapFileCommentRegex` property in the default export of `convert-source-map` package. Maybe we could use this.